packages/stack/proxy/src/index.ts

```diff
@@ -49,9 +49,9 @@ export default class ProxyManager {
       }
     });
 
-    this.events.on("blockchain:stopped", async (clientName: string, node?: string) => {
+    this.events.on("blockchain:stopped", (clientName: string, node?: string) => {
       this.ready = false;
-      await this.stopProxy();
+      this.stopProxy();
```

Comment:
`stopProxy` is actually not async so no need to await here

Reply:
There is a possibility that `this.server.stop()`, executed inside of `this.stopProxy()` requires a callback or does not stop the proxy immediately. Let's find out about this before approving the PR. Will mark as "changes required" for now to prevent merging while this is investigated.